### PR TITLE
for show vote the user needs a counted assurance

### PR DESCRIPTION
**Pull request by lbarjak**

### CODE REVIEW

File: src/test/java/org/rulez/demokracia/pdengine/ForShowVoteTheUserNeedsACountedAssurance.java [hunk 1]
```diff
@@ -30,9 +23,10 @@ public void create_creates_a_vote_with_countedAssurances() throws ReportedExcept
 		Vote vote = getTheVote();
 		vote.adminKey = "anon";
 		vote.countedAssurances.add("magyar");
-		if(!(vote.adminKey.equals("anon") 
-				&& vote.neededAssurances.containsAll(countedAssurances))) {
-			fail("anon admin key without countedAssurances");
-		}	
+
```
Comment (magwas) [inline src/test/java/org/rulez/demokracia/pdengine/ForShowVoteTheUserNeedsACountedAssurance.java:40]: I don't see the relevance to the issue.

**Comment by magwas** [comment]
uppsz, a
adminKey != vote.adminKey
helyett
! adminKey.equals(vote.adminkey)
kellett volna.

**Comment by magwas** [comment]
there are code coverage issues.

**Comment by magwas** [comment]
the if in line 22 is unnecessary (I have no idea how that got through)
there is no testcase for showVote where adminkey="user", and the assurances are checked.
It seems that there is such a case in edemo/develop though, and the conflict is because that.

**Comment by magwas** [comment]
It seems that there is a problem with measuring code coverage for assertThrows.
Here is the relevant stack owerflow question: https://stackoverflow.com/questions/55282580/code-coverage-does-not-seem-to-work-for-assertion-employing-functionalinterface
So you just have to resolve the conflict, and get rid of the unnecessary if.

**Comment by magwas** [comment]
Van szükséged segítségre?

**Comment by lbarjak** [comment]
Van igen, nem sikerült a fork nyolcadik frissítése, letöröltem (lehet, hogy ezt nem kellett volna) és újból forkoltam, most nem látom a branch-et.

**Comment by magwas** [comment]
Ez a pull request nem folytatható. A kód ami benne van megvan neked?
Ha igen, akkor nyiss új pull requestet, ha nem, akkor megpróbáljuk megmenteni innen.

**Pull request closed without merging**